example/zping/cmd/client.go

```diff
@@ -0,0 +1,259 @@
+/*
```

Comment:
you can grab a good header from most other files: https://github.com/openziti/sdk-golang/blob/main/ziti/ziti.go#L1-L15